proposal/active_span_source.py

```diff
@@ -20,3 +23,23 @@ def make_active(self, span):
 
     def active_span(self):
         return None
+
+
+class ThreadActiveSpanSource():
```

Comment:
Should inherit from `BaseActiveSpanSource`? Doesn't change much in Python, just to show it inherits the interface.